pkg/controller/component/deployment.go

```diff
@@ -2,7 +2,7 @@ package component
 
 import (
 	component "halkyon.io/api/component/v1beta1"
-	"halkyon.io/operator/pkg/controller/framework"
+	"halkyon.io/operator-framework"
```

Comment:
What is the purpose of this new project `operator-framework` ?

Reply:
As its name imply it's a framework to make it easier to create operators.